src/icons/Manage.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: string
+  testID?: string
+  size?: number
+}
+
+function Manage({ color = colors.primary, testID = 'Manage', size = 24 }: Props) {
+  return (
+    <Svg width={size} height={size} viewBox="0 0 24 24" fill="none" testID={testID}>
```

Comment:
```suggestion
    <Svg width={size} height={size} viewBox="0 0 20 20" fill="none" testID={testID}>
```

Reply:
Since the original icon was 20x20 we need the view box to match. When the viewBox is bigger than the icon the SVG will displays the icon + whitespace as seen to the right and bottom.